src/assets/wise5/authoringTool/project-authoring/project-authoring.component.html

```diff
@@ -75,32 +75,51 @@
   </button>
 </div>
 <div class="all-nodes-div">
-  <div *ngFor="let lesson of lessons">
+  <div *ngIf="lessons.length === 0" fxLayout="row" fxLayoutAlign="start center">
+    <div i18n>There are no lessons</div>
+    <add-lesson-button [active]="true"></add-lesson-button>
+  </div>
+  <div *ngFor="let lesson of lessons" fxLayout="row">
     <project-authoring-lesson
       [lesson]="lesson"
       (selectNodeEvent)="selectNode($event)"
       [showPosition]="true"
       [projectId]="projectId"
       [expanded]="lessonIdToExpanded()[lesson.id]"
       (onExpandedChanged)="onExpandedChanged($event)"
+      fxFlex
     ></project-authoring-lesson>
+    <add-lesson-button
+      class="add-lesson-button-next-to-lesson"
+      [active]="true"
+      [lessonId]="lesson.id"
+    ></add-lesson-button>
   </div>
   <div>
     <h6 class="unused-header" i18n>Unused Lessons</h6>
-    <div *ngIf="inactiveGroupNodes.length === 0" i18n>There are no Unused Lessons</div>
-    <div *ngFor="let inactiveGroupNode of inactiveGroupNodes">
+    <div *ngIf="inactiveGroupNodes.length === 0" fxLayout="row" fxLayoutAlign="start center">
+      <div i18n>There are no unused lessons</div>
+      <add-lesson-button [active]="false"></add-lesson-button>
```

Comment:
Unnecessary attribute, remove?
```suggestion
      <add-lesson-button></add-lesson-button>
```